pom.xml

```diff
@@ -0,0 +1,183 @@
+		<module>spring-cloud-gcp-pubsub</module>
+		<module>spring-cloud-gcp-core</module>
+		<module>spring-cloud-gcp-storage</module>
+	</modules>
+
+	<dependencyManagement>
+		<dependencies>
+			<dependency>
+				<groupId>org.springframework.cloud</groupId>
+				<artifactId>spring-cloud-gcp-dependencies</artifactId>
+				<version>${project.version}</version>
+				<type>pom</type>
+				<scope>import</scope>
+			</dependency>
+		</dependencies>
+	</dependencyManagement>
+
+	<dependencies>
+		<dependency>
+			<groupId>junit</groupId>
+			<artifactId>junit</artifactId>
+			<scope>test</scope>
+		</dependency>
+		<dependency>
+			<groupId>org.slf4j</groupId>
+			<artifactId>slf4j-api</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>org.mockito</groupId>
+			<artifactId>mockito-core</artifactId>
+			<scope>test</scope>
+		</dependency>
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter-test</artifactId>
+			<scope>test</scope>
+		</dependency>
+
+	</dependencies>
+
+	<build>
+		<pluginManagement>
+			<plugins>
+				<plugin>
+					<artifactId>maven-compiler-plugin</artifactId>
+					<configuration>
+						<compilerArgs>
+							<!-- Enable all warnings -->
+							<compilerArg>-Xlint:all</compilerArg>
+							<!-- Disable options warning because we will have differences between the compiler and source code level-->
+							<compilerArg>-Xlint:-options</compilerArg>
+							<!-- Disable serialversionuid warnings -->
+							<compilerArg>-Xlint:-serial</compilerArg>
+							<!--compilerArg>-Werror</compilerArg-->
+						</compilerArgs>
+					</configuration>
+				</plugin>
+				<plugin>
+					<groupId>org.apache.maven.plugins</groupId>
+					<artifactId>maven-surefire-plugin</artifactId>
+					<configuration>
+						<argLine>-Xms512m -Xmx512m</argLine>
+					</configuration>
+				</plugin>
+			</plugins>
+		</pluginManagement>
+	</build>
+
+	<reporting>
+		<plugins>
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-javadoc-plugin</artifactId>
+				<configuration>
+					<links>
+						<link>http://docs.oracle.com/javase/7/docs/api/</link>
+						<link>http://docs.oracle.com/javaee/7/api/</link>
+						<link>http://fasterxml.github.com/jackson-core/javadoc/2.0.0/</link>
+						<link>http://docs.spring.io/spring/docs/4.1.x/javadoc-api/</link>
+					</links>
+					<author>true</author>
+					<header>${project.name}</header>
+				</configuration>
+			</plugin>
+		</plugins>
+	</reporting>
+
+	<profiles>
+		<profile>
+			<id>integration-test</id>
+			<activation>
+				<property>
+					<name>els.config.dir</name>
+				</property>
+			</activation>
+			<modules>
+				<module>spring-cloud-aws-integration-test</module>
+			</modules>
+		</profile>
+		<profile>
+			<id>spring</id>
+			<repositories>
+				<repository>
+					<id>spring-snapshots</id>
+					<name>Spring Snapshots</name>
+					<url>https://repo.spring.io/libs-snapshot-local</url>
```

Comment:
we could activate the `spring` profile by default, but this works now:

```
mvn -Pspring clean package
```

(although my current build from scratch had a test failure because of an assumption that credentials are defined)

Also, we should go ahead and add the mvn wrapper (in a separate PR would be fine).